prow_config.yaml

```diff
@@ -29,48 +29,23 @@ workflows:
       installIstio: true
       workflowName: deployapp-istio
   - py_func: kubeflow.kubeflow.ci.kfctl_e2e_workflow.create_workflow
-    name: kfctl-go-iap-endpoint
+    name: kfctl-go-iap
     job_types:
+      - presubmit
       - postsubmit
       - periodic
     include_dirs:
       - bootstrap/*
-      - deployment/*
-      - dependencies/*
-      - kubeflow/*
-      - testing/*
-    kwargs:
-      use_basic_auth: false
-      test_endpoint: true
-      config_path: https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_gcp_iap.yaml
-  - py_func: kubeflow.kubeflow.ci.kfctl_e2e_workflow.create_workflow
-    component: kfctl_go_test
-    name: kfctl-go-iap-istio
-    job_types:
-      - presubmit
-    include_dirs:
-      - bootstrap/*
-      - kubeflow/*
-      - testing/*
-    kwargs:
-      use_basic_auth: false
-      config_path: https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_gcp_iap.yaml
-  # If the kfctl_e2e_workflow.py was modified then on presubmit we also want to run with test_endpoint true
-  # to ensure that test still pasess
-  - py_func: kubeflow.kubeflow.ci.kfctl_e2e_workflow.create_workflow
-    component: kfctl_go_test
-    name: kfctl-gcp-e2e-changed
-    job_types:
-      - presubmit
-    include_dirs:
-      - bootstrap/*
-      - kubeflow/*
       - testing/*
       - py/kubeflow/kubeflow/ci/kfctl_e2e_workflow.py 
     kwargs:
       use_basic_auth: false
+      # Run build and then apply rather than just apply
+      build_and_apply: true
```

Comment:
Since kfctl apply only is more common (I suppose?), should we set this to false?

Reply:
This flag itself may not be needed after some more refactoring and re-organization of e2e tests, but its not high on priority I suppose.

Reply:
My thinking is that we get more coverage by doing

```
kfctl build ...
kfctl apply
```

then just doing 

```
kfctl apply
```

My expectation is also that we will have more manual testing of just running

```
kfctl apply
```

So I'd expect if we just run `kfctl apply` has a bug we are more likely to know about it.